lib/galaxy/config/schemas/config_schema.yml

```
@@ -3912,3 +3912,19 @@ mapping:
         default: 86400
         desc: |
           The interval in seconds between attempts to delete all expired notifications from the database (every 24 hours by default). Runs in a Celery task.
+
+      help_forum_api_url:
+        type: str
+        required: false
+        default: https://help.galaxyproject.org/
+        per_host: true
+        desc: |
+          The URL pointing to the Galaxy Help Forum API base URL. The API must be compatible with Discourse API (https://docs.discourse.org/).
+      
+      help_forum_tool_panel_integration_enabled:
```

```suggestion
      enable_help_forum_tool_panel_integration:
```

I know we got one more case of _enabled in `static_enabled`, but the rest of the options are `enable_`